docs/families/WannaCry.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+# TODO - uncomment out the following. Do not change the name.
+parent: Ransomware Families
+
+# TODO - uncomment the below and change "Black Cat" to the new name. This will be shown in the menu.
+title: WannaCry
+
+# TODO - uncomment below and change "blackcat" to the new url. It should be all lower case and no spaces
+permalink: /families/wannacry/
+---
+
+| Category | Answer | References | 
+| ----------- | ----------- | | 
+|**Actors** | | |
+|First Observed | May 2017 | [^1] |
+|Threat Actors | North Korea | [^2]|
+|**Environment** | | |
+|Platforms | Windows | [^3]|
+|**Artifacts** | | |
+|Extensions | .wannacry .wcry .Wnry .wncry | [^4] [^5] [^6] [^7]|
+|Ransomware Notes | info.hta | [^8] |
+|Services It Disables | | |
+|Other Observables | Various YARA rules in existence | [^9] |
+|**Automation** | | |
+|Initial Access | Yes, EternalBlue| [^10] |
+|Privilege Escalation | No, not needed. Worm-like| |
```

Comment:
Is that the right answer for this? Seems like it would be soething like "No, not needed because EternalBlue provides admin".  If that is true. I forget.